wallet/src/Pos/Wallet/Web/Pending/Submission.hs

```diff
@@ -136,11 +136,15 @@ submitAndSavePtx PtxSubmissionHandlers{..} enqueue ptx@PendingTx{..} = do
     minorError desc e = do
         reportError desc e ", but was given another chance"
         pshOnMinor e
-    nonReclaimableError accepted e = do
+    nonReclaimableError e = do
         reportError "fatal" e ""
-        pshOnNonReclaimable accepted e
+        pshOnNonReclaimable e
 
     reportError desc e outcome =
         logInfoS $
         sformat ("Transaction #"%build%" application failed ("%shown%" - "
                 %stext%")"%stext) _ptxTxId e desc outcome
+    reportSubmitted ack =
+        logDebug $
+        sformat ("submitAndSavePtx: transaction submitted with confirmation?: "
```

Comment:
This one was also added externally as it's not part of https://github.com/input-output-hk/cardano-sl/pull/2270 .

Reply:
https://github.com/input-output-hk/cardano-sl/commit/66b487addc70adf5ca61bba6abbce87df2efa2a3 but required for https://github.com/input-output-hk/cardano-sl/commit/fa3d120b4121dd8b72f3fd3ec629633785da7795